README.rst

```diff
@@ -45,11 +45,9 @@ Main Advantages
 - Fully multi-platform, and uses the OS support to load the dynamic libraries,
   thus ensuring full compatibility.
 - Correctly bundles the major Python packages such as numpy, PyQt5,
-  PySide2, Django, wxPython, matplotlib and others out-of-the-box.
+  PySide2, NumPy, wxPython, matplotlib and others out-of-the-box.
```

Comment:
While numpy *is* a major package, we probably don't have to list it twice? :)

Perhaps PyQt6 and PySide6 can be added to the list instead.

Reply:
woopsie